LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift

```diff
@@ -11,8 +11,9 @@ import HealthKit
 import LoopKit
 
 public struct CorrectionRangeOverridesEditor: View {
```

Comment:
There's an issue with the confirmation alert, where if you set the premeal range outside of guardrails (to trigger the pop-up warning), then the same “out of guardrails” warning on the workout range is displayed even if it’s within bounds (the relevant function is `confirmationAlert()`)

Reply:
I think I've addressed it with https://github.com/tidepool-org/LoopKit/pull/206/commits/0220219776506cc0a3276cfe7a71d8f5ceef147a, but now I can't trip the workout range guardrail... 🤷

Reply:
I can't find a way to trip the workout range guardrail to begin with - I think that's a bug that, while related to your PR, shouldn't block it.

Reply:
OK looks like that's actually the intended design - no high warning, and low warning at 70 mg/dl (see https://tidepool.atlassian.net/browse/LOOP-1318). FWIW I think there _should_ be a low warning (70 is crazy for exercise), but that's not in the scope of this ticket and I have a feeling the revised guardrails will change that